2-collectors/thrift-schemas/collector-payload-1/src/main/thrift/collector-payload.thrift

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2014 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0, and
+ * you may not use this file except in compliance with the Apache License
+ * Version 2.0.  You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the Apache License Version 2.0 is distributed on an "AS
+ * IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the Apache License Version 2.0 for the specific language
+ * governing permissions and limitations there under.
+ */
+
+namespace java com.snowplowanalytics.snowplow.CollectorPayload.thrift.model1
+
+struct CollectorPayload {
+  31337: string schema
+  10: optional string querystring
+  20: string collector
+  30: string encoding
+  40: optional string hostname
+  50: i64 timestamp
+  60: string ipAddress
+  70: optional string userAgent
+  80: optional string refererUri
+  90: optional list<string> headers
+  100: optional string networkUserId
+  110: string path
+  120: optional string contentType
+  130: optional string body
+}
```

Comment:
Hey @fblundun - can we come up with a more human-readable ordering/numbering/grouping of these fields please. Anyone coming to this cold won't be able to discern any pattern to this structure... A more coherent ordering would look something like:

    Required fields which are intrinsic properties of HTTP
    Required fields which are Snowplow-specific
    Optional fields which are intrinsic properties of HTTP
    Optional fields which are Snowplow-specific

It would be worth giving each of these four clusters their own block (100, 200, 300, 400 kind of thing).

Also - it seems odd that path is required. I think path always being set is just an artifact of the Scala Stream Collector implementation - certainly path can be null in our CloudFront and Clojure Collectors.